modules/test-structure/save_test_data.go

```diff
@@ -282,3 +282,28 @@ func CleanupTestData(t *testing.T, path string) {
 		logger.Logf(t, "%s does not exist. Nothing to cleanup.", path)
 	}
 }
+
+// CleanupTestDataFolder cleans up the .test-data folder inside the given folder.
+// If deleteParent is true, CleanupTestDataFolder will also delete the given folder.
+func CleanupTestDataFolder(t *testing.T, path string, deleteParent bool) {
+	if !deleteParent {
+		path = filepath.Join(path, ".test-data")
+	}
+	cleanupTestDataFolder(t, path)
+}
+
+// cleanupTestDataFolder deletes the given folder.
+func cleanupTestDataFolder(t *testing.T, path string) {
```

Comment:
Please check out these guidelines about how we handle errors in Terratest: https://github.com/gruntwork-io/terratest/#error-handling

Reply:
My bad, I will update the code accordingly to follow the error handling conventions.